src/Anonymization/Anonymizer/Options.php

```diff
@@ -51,4 +55,75 @@ public function toDisplayString(): string
             \array_values($this->options),
         ));
     }
+
+    public function getString(string $name, string $default = null, bool $required = false): ?string
+    {
+        $value = $this->get($name, $default, $required);
+        if ($value === null || is_scalar($value) || $value instanceof \Stringable) {
```

Comment:
Are you considering null as an empty string ? You should always start with checking if null and not required then return null.

Reply:
I wonder if we should consider `null` as empty or not ?

(I updated tests in order to accept `null` values but not sure if it is a good choice..)

Reply:
`null` handling added

Reply:
It seems cleaner to me, nice.